src/esi.cls.php

```diff
@@ -760,7 +763,12 @@ public function sub_widget_block($instance, $widget, $args)
 	public function sub_admin_bar_block()
 	{
 		global $wp_admin_bar;
+		static $rendered = false;
```

Comment:
Don't define static here, plz define it in class level. Otherwise others won't be aware of this var and may cause conflict.